app/Controller/PagesController.php

```diff
@@ -74,3 +120,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
```

Comment:
__[staging][code_sniffer]__
* Missing function doc comment
* Double space found
* Opening brace should be on a new line